lwallet/Command.hs

```diff
@@ -111,6 +113,20 @@ switch = lexeme $ positive $> True <|>
     positive = text "+" <|> text "y" <|> text "yes"
     negative = text "-" <|> text "n" <|> text "no"
 
+stakeholderId :: Parser StakeholderId
+stakeholderId = lexeme $ do
+    str <- many1 alphaNum
+    case decodeTextAddress (toText str) of
```

Comment:
I'm trying to write with `hashHexF` ... `addressHash pk`, then read with `decodeAbstractHash`. It results in error:
```
 StakeholderId: d708352cbdad7e4e579df1d6e3482ac002121fcd2f085790f6b1da1a
> send-distr 0 jbRJqzVmBoUNqAQBykQsAeFC12GEa4ysPqiGidKSrNzU5shUzxdPcVMRhWPN 33333 6056d5a781311a76d5cd7b4f43823de687bb0cba5c4482bf4579fc9d 33333
cardano-wallet: decode hash error: decodeFull failed for AbstractHash Blake2b_224 PublicKey: DeserialiseFailure 0 "expected bytes"
CallStack (from HasCallStack):
  error, called at src/Debug.hs:43:11 in universum-0.4.3-5s0jbvCPrRj9dDakAh9rIJ:Debug
```